data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/beacon/GetPeersScore.java

```diff
@@ -0,0 +1,58 @@
+package tech.pegasys.teku.beaconrestapi.handlers.tekuv1.beacon;
+
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.CACHE_NONE;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.RES_INTERNAL_ERROR;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.RES_OK;
+import static tech.pegasys.teku.beaconrestapi.RestApiConstants.TAG_V1_NODE;
+
+import io.javalin.core.util.Header;
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+import io.javalin.plugin.openapi.annotations.HttpMethod;
+import io.javalin.plugin.openapi.annotations.OpenApi;
+import io.javalin.plugin.openapi.annotations.OpenApiContent;
+import io.javalin.plugin.openapi.annotations.OpenApiResponse;
+import org.jetbrains.annotations.NotNull;
+import tech.pegasys.teku.api.DataProvider;
+import tech.pegasys.teku.api.NetworkDataProvider;
+import tech.pegasys.teku.api.response.v1.node.GetPeerScoresResponse;
+import tech.pegasys.teku.api.response.v1.node.Peer;
+import tech.pegasys.teku.api.response.v1.node.PeerScore;
+import tech.pegasys.teku.api.response.v1.node.PeersResponse;
+import tech.pegasys.teku.provider.JsonProvider;
+
+import java.util.List;
+
+public class GetPeersScore implements Handler {
+    public static final String ROUTE = "/teku/v1/nodes/peer_scores";
```

Comment:
If we want to file this under a "node" endpoint (which makes sense), we should create a node package and move this class there: `tech.pegasys.teku.beaconrestapi.handlers.tekuv1.node`

Reply:
I've created the new package. Thank you!